camsys/transam_cpt
# Transam dependencies

* Make travis get latest versions of depended-on engines
* Make sure Gemfile.lock is not committed
* Remove engines from gemspec, since they are still in development
